docs/draft/webview.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Dapp browser API usage
+version: 0.1.0
+status: Draft
+authors: 
+---
+
+# Dapp browser API usage
+    
+##  Table of Contents
+
+1. [Abstract](#abstract)
+2. [Definitions](#definitions)
+3. [Implementation](#implementation)
+4. [Usage](#usage)
+    * [Properties](#properties)
+        * [`isStatus`](#isStatus)
+        * [`status`](#status)
+    * [Methods](#methods)
+        * [`enable`](#enable)
+        * [`isConnected`](#isConnected)
+        * [`scanQRCode`](#scanQRCode)
+        * [`send`](#send)
+        * [`sendAsync`](#sendAsync)
+        * [`sendSync`](#sendSync)
+5. [Compatibility](#compatibility)
+6. [Changelog](#changelog)
+7. [Acknowledgements](#acknowledgements)
+8. [Copyright](#copyright)
+9. [Footnotes](#footnotes)
+
+## Abstract
+
+Status exposes an Ethereum Provider object (`window.ethereum`) to JS code running inside the browser. It also provides bidirectional communication between JS code and Status.
+
+## Definitions
+
+| Term       | Description                                                                         |
+|------------|-------------------------------------------------------------------------------------|
+| **Webview**   | React Native wrapper for a platform-specific browser core implementation.                                        |
```

Comment:
Does it have to be a RN wrapper, or could it be generalized? On the spec level, now current app level

Reply:
Removed the mention of RN wrapper and modified wording elsewhere, so that it's more generic.